cmd/jwt-provider/main.go

```diff
@@ -0,0 +1,45 @@
+package jwt_provider
+
+import (
+	"context"
+
+	"github.com/forta-network/forta-core-go/clients/health"
+	"github.com/forta-network/forta-node/config"
+	"github.com/forta-network/forta-node/healthutils"
+	"github.com/forta-network/forta-node/services"
+	jwt_provider "github.com/forta-network/forta-node/services/jwt-provider"
+)
+
+func initJWTProvider(cfg config.Config) (*jwt_provider.JWTProvider, error) {
+	return jwt_provider.NewJWTProvider(cfg)
+}
+
+func initServices(ctx context.Context, cfg config.Config) ([]services.Service, error) {
+	jwtProvider, err := initJWTProvider(cfg)
+	if err != nil {
+		return nil, err
+	}
+
+	return []services.Service{
+		health.NewService(
+			ctx, "", healthutils.DefaultHealthServerErrHandler,
+			health.CheckerFrom(summarizeReports, jwtProvider),
+		),
+		jwtProvider,
+	}, nil
+}
+
+func summarizeReports(reports health.Reports) *health.Report {
```

Comment:
I'd consider any failure in this service to be technical. It's nice that we are checking the API status and using a tracker to keep the latest error, then return it when requested. But this "summary" feature is mainly for enhancing the node operator experience. I really recommend seeing the one in `cmd/scanner`.

When we put this error in to summaries, it becomes one of the first things the node operator sees in the `forta status` output and this error is hard for them to understand. Let's send the first arg to `CheckerFrom` as `nil` so we disable any summaries from this service and it would be visible in telemetry and when operators use one of the `forta status` flags. Thanks!

Reply:
this one makes sense, added